packages/studio-plugin/src/writers/StudioSourceFileWriter.ts

```diff
@@ -77,9 +79,13 @@ export default class StudioSourceFileWriter {
     });
     this.sourceFile.fixMissingImports();
     cssImports?.forEach((importSource) => {
-      this.sourceFile.addImportDeclaration({
-        moduleSpecifier: importSource,
-      });
+      const moduleSpecifier = upath.isAbsolute(importSource)
```

Comment:
is this check needed because calling `getImportSpecifierWithExtension` for an npm package would result in an incorrect import path?

Reply:
yeah, it traces it back as a relative path from the import source instead of as a normal node module import